lgsm/functions/update_factorio.sh

```diff
@@ -9,6 +9,18 @@ local commandname="UPDATE"
 local commandaction="Update"
 local function_selfname="$(basename "$(readlink -f "${BASH_SOURCE[0]}")")"
 
+fn_update_factorio_clear_tmp(){
+	echo -e "Clear tmpdir...\c"
+	fn_script_log "Directory ${tmpdir} cleared"
```

Comment:
Past tense when deletion hasn't happened yet - may be confusing. I suggest this is turn to "clearing" and the if statement below changed to print if it was cleared successfully.